tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py

```diff
@@ -0,0 +1,384 @@
+import logging
+import time
+from concurrent.futures import ThreadPoolExecutor
+import pytest
+
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources import pod as pod_helpers
+from ocs_ci.framework.testlib import tier1, ignore_leftovers, ManageTest
+from tests.manage.z_cluster import cluster_exp_helpers
+from ocs_ci.ocs import constants
+from tests.manage.mcg.helpers import s3_io_create_delete, obc_io_create_delete
+from ocs_ci.ocs import node
+from ocs_ci.ocs import cluster as cluster_helpers
+from ocs_ci.ocs.resources import storage_cluster
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.framework import config
+from tests.manage.z_cluster.pvc_ops import test_create_delete_pvcs
+
+
+# Add capacity needs to happen with background operations/IOs in progress. In order to do this we use threads. Each
```

Comment:
with class for checking background ops available now, we do not need the comments from line 22 to 29.